RecoBTag/SecondaryVertex/python/candidateNegativeCombinedSecondaryVertexComputer_cfi.py

```diff
@@ -0,0 +1,11 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoBTag.SecondaryVertex.candidateCombinedSecondaryVertexComputer_cfi import *
+
+candidateNegativeCombinedSecondaryVertexComputer = candidateCombinedSecondaryVertexComputer.clone()
+candidateNegativeCombinedSecondaryVertexComputer.vertexFlip = True
```

Comment:
Honestly, I don't see the point of doing this modification since the following 4 parameters will still have to be explicitly overwritten. Otherwise, I would have done it. There are also some old configs that would require similar updates but updating many config files for mostly aesthetic reasons is just calling for trouble since unintended bugs could creep in ;)